docs/release-v2.md

```diff
@@ -0,0 +1,167 @@
+
+### Write release description for non-developers
+
+Turtles release notes are automatically generated based on the commits that are published with the new version.
+
+These commit messages are usually hard to understand for those that are not very familiar with the project and we must add a "human-readable" description of what the release brings at the beginning of release notes. [This](https://github.com/rancher/turtles/releases/tag/v0.13.0) can be used for reference.
+
+## Versioning
+
+Rancher Turtles follows [semantic versioning](https://semver.org/) specification.
+
+Example versions:
+- Pre-release: `v0.4.0-alpha.1`
+- Minor release: `v0.4.0`
+- Patch release: `v0.4.1`
+- Major release: `v1.0.0`
+
+With the v0 release of our codebase, we provide the following guarantees:
+
+- A (*minor*) release CAN include:
+  - Introduction of new API versions, or new Kinds.
+  - Compatible API changes like field additions, deprecation notices, etc.
+  - Breaking API changes for deprecated APIs, fields, or code.
+  - Features, promotion or removal of feature gates.
+  - And more!
+
+- A (*patch*) release SHOULD only include backwards compatible set of bugfixes.
+
+### Backporting
+
+Any backport MUST not be breaking for either API or behavioral changes.
+
+It is generally not accepted to submit pull requests directly against release branches (release/X). However, backports of fixes or changes that have already been merged into the main branch may be accepted to all supported branches:
+
+- Critical bugs fixes, security issue fixes, or fixes for bugs without easy workarounds.
+- Dependency bumps for CVE (usually limited to CVE resolution; backports of non-CVE related version bumps are considered exceptions to be evaluated case by case)
+- Cert-manager version bumps (to avoid having releases with cert-manager versions that are out of support, when possible)
+- Changes required to support new Kubernetes versions, when possible. See supported Kubernetes versions for more details.
+- Changes to use the latest Go patch version to build controller images.
+- Improvements to existing docs (the latest supported branch hosts the current version of the book)
+
+**Note:** We generally do not accept backports to Rancher Turtles release branches that are out of support.
+
+#### Automation to create backports
+
+There is automation in place to create backport PRs when a comment of a certain format is added to the original PR:
+
+```
+/backport <milestone> <target_branch>
+```
+
+For example, adding the comment `/backport v2.12.1 release/v0.25` to a PR will result in creating a new backport PR against release branch `release/v0.25` with a patch from the PR that the comment was added to and a milestone set to `v2.12.1`. [Here](https://github.com/rancher/turtles/pull/1782) is an example of a backport PR generated through this automation.
```

Comment:
we can just refer to https://github.com/rancher/turtles/blob/main/docs/release-automation-workflows.md#backbackport-pr-automation instead

Reply:
yeah, looks like a typo 😢